adders/tailwindcss/config/adder.ts

```diff
@@ -92,7 +92,9 @@ export const adder = defineAdderConfig({
             content: ({ js, html }) => {
                 js.imports.addEmpty(js.ast, "../app.css");
                 const slot = html.element("slot");
-                html.ast.childNodes.push(slot);
+                if (html.ast.childNodes.length === 0) {
```

Comment:
I imagine this would be triggered in an empty layout or script-only layout, which I suppose is fine, but I kind of expected the check for no file existing would be different than the check for empty file. It seems like we might not be able to tell those two cases apart

Reply:
> It seems like we might not be able to tell those two cases apart

that's correct. within the context of the `content` hook, there's no easy way to tell if the file is being created or not

Reply:
> I imagine this would be triggered in an empty layout or script-only layout, which I suppose is fine

testing this now, it doesn't trigger if there's a script tag inside since that's considered a child node. If the layout is empty of text, _then_ it'll add the slot